### PR TITLE
Updated table date formatting and alignment.

### PR DESCRIPTION
**Pull request by dsjen**
- Added table pagination.
- Moved download button to section heading.
- Changed color of download buttons to gray.
- Aligned numbers in tables to the right.
- Added moment.js package for date formatting.

data-tables-formatted-dates

### CODE REVIEW

**Comment by dsjen** [comment]
@marcotuts @clintonb @rocha

File: analytics_dashboard/static/js/views/data-table-view.js [hunk 1]
```diff
@@ -41,26 +47,52 @@ define(['dataTablesBootstrap', 'jquery', 'underscore', 'views/attribute-listener
...
-                        data: this.model.get(self.options.modelAttribute),
-                        columns: dtColumns
-                    };
+                    paging: true,
+                    info: false,
+                    filter: false,
+                    data: self.model.get(self.options.modelAttribute),
+                    columns: self._buildColumns($row),
+                    columnDefs: self._buildColumnDefs(),
+                    // this positions the "length changing" control to the bottom
```
Comment (clintonb) [inline analytics_dashboard/static/js/views/data-table-view.js:91]: By "length changing", do you mean "pagination"?

**Comment by clintonb** [comment]
Looks fine, although we will need to add i18n support: http://www.datatables.net/plug-ins/i18n/. This can be done in a separate issue.

File: analytics_dashboard/static/js/views/data-table-view.js (hunk 1 above)
Comment (dsjen) [inline analytics_dashboard/static/js/views/data-table-view.js:91]: "Length changing" refers to control for selecting the number of rows per page displayed in the table at a time.  It's the somewhat odd nomenclature that datatables uses ('l' - Length changing), http://legacy.datatables.net/usage/options.

**Comment by rocha** [comment]
:rocket: Nice!

**Comment by dsjen** [comment]
Great!  i18n support logged in https://openedx.atlassian.net/browse/AN-3218.

**Pull request merged**